### PR TITLE
Add proposed announcements on Observation/Thoughts and Rust Walkthroughs

### PR DESCRIPTION
**Pull request by rsdlt**

### CODE REVIEW

**Comment by andrewpollack** [comment]
Thank you for the post! 

The "Basic Operator Overloading with Traits" is great to accept as is.

Unfortunately, "Ruxel - Building a Ray Tracer with Rust (Prelude)" is a little lighter on implementation and walkthrough details to include in TWiR. That said, we're excited to see this series continue, and review future editions in this series

File: draft/2022-08-10-this-week-in-rust.md [hunk 1]
```diff
@@ -36,9 +36,9 @@ and just ask the editors to select the category.
 ### Project/Tooling Updates
 
 ### Observations/Thoughts
-
+[Basic Operator Overloading with Traits](https://rsdlt.onrender.com/posts/welcome-blog-rust-technology-development-programming-language/)
 ### Rust Walkthroughs
-
+[Ruxel - Building a Ray Tracer with Rust (Prelude)][series](https://rsdlt.onrender.com/posts/ruxel-ray-tracer-project-first-update-rust-programming-development/)
```
Comment (andrewpollack) [inline draft/2022-08-10-this-week-in-rust.md:41]:
```suggestion
```

**Review by andrewpollack — CHANGES_REQUESTED** [review]

**Comment by rsdlt** [comment]
Thanks Andrew! Excited to be part of this great community.

**Comment by andrewpollack** [comment]
Looks great -- merging!

File: draft/2022-08-10-this-week-in-rust.md [hunk 2]
```diff
@@ -36,9 +36,8 @@ and just ask the editors to select the category.
 ### Project/Tooling Updates
 
 ### Observations/Thoughts
-
```
Comment (andrewpollack) [inline draft/2022-08-10-this-week-in-rust.md:39]: Github is having a tough time resolving the merge conflict...  can this blank line be restored? Thank you!

File: draft/2022-08-10-this-week-in-rust.md [hunk 3]
```diff
@@ -36,9 +36,8 @@ and just ask the editors to select the category.
...
+[Basic Operator Overloading with Traits](https://rsdlt.onrender.com/posts/welcome-blog-rust-technology-development-programming-language/)
 ### Rust Walkthroughs
-
```
Comment (andrewpollack) [inline draft/2022-08-10-this-week-in-rust.md:41]: Github is having a tough time resolving the merge conflict...  can this blank line be restored as well? Thank you!

**Review by andrewpollack — CHANGES_REQUESTED** [review]

**Review by rsdlt — COMMENTED** [review]
Sure!

**Comment by andrewpollack** [comment]
Fantastic -- thank you!! Merging now

**Pull request merged**